arduino-ide-extension/src/browser/arduino-preferences.ts

```diff
@@ -174,6 +174,14 @@ export const ArduinoConfigSchema: PreferenceSchema = {
       ),
       default: 'https://auth.arduino.cc/login#/register',
     },
+    'arduino.cli.daemon.debug': {
+      type: 'boolean',
+      description: nls.localize(
+        'arduino/preferences/cli.daemonDebug',
+        "Enable debug logging of the gRPC calls to the Arduino CLI. It's false by default."
```

Comment:
@kittaakos let's clarify that a restart of the IDE is needed. 
```suggestion
        "Enable debug logging of the gRPC calls to the Arduino CLI. A restart of the IDE is needed for this setting to take effect. It's false by default."
```